c++/src/kj/async-test.c++

```diff
@@ -26,7 +26,7 @@
 #include "thread.h"
 
 // TODO(later): Refactor this more cleanly so that the KJ library itself defines this.
-#if !__BIONIC__ && !KJ_NO_EXCEPTIONS
+#if KJ_USE_FIBERS && !KJ_NO_EXCEPTIONS
 #define KJ_FIBERS_AVAILABLE 1
```

Comment:
Can we just remove this define entirely and replace `KJ_FIBERS_AVAILABLE` with `KJ_USE_FIBERS`? (See the TODO, above.)

Reply:
Done.